### PR TITLE
Update license header

### PR DESCRIPTION
**Pull request by weiqingy**
This PR is to update license header.

### CODE REVIEW

**Comment by weiqingy** [comment]
cc: @bikassaha @dongjoon-hyun

File: core/src/test/scala/org/apache/spark/sql/AvroSourceSuite.scala [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * File modified by Hortonworks, Inc. Modifications are also licensed under
+ * the Apache Software License, Version 2.0.
+ */
+
 package org.apache.spark.sql
```
Comment (dongjoon-hyun) [inline core/src/test/scala/org/apache/spark/sql/AvroSourceSuite.scala]: Oh, this suite is missing the license completely before.

Comment (weiqingy) [reply core/src/test/scala/org/apache/spark/sql/AvroSourceSuite.scala]: @dongjoon-hyun yes, I should have found this issue before. :(

Comment (dongjoon-hyun) [reply core/src/test/scala/org/apache/spark/sql/AvroSourceSuite.scala]: Later, you can add rat check and trigger on Travis CI automatically.

Comment (weiqingy) [reply core/src/test/scala/org/apache/spark/sql/AvroSourceSuite.scala]: Ok. Will do that. Thanks, Dongjoon.

**Comment by dongjoon-hyun** [comment]
Hi, @weiqingy .
Is there anything for you not to merge this?

**Comment by weiqingy** [comment]
@dongjoon-hyun @bikassaha I have updated the license header for avro file(see [the commit](https://github.com/hortonworks-spark/shc/pull/182/commits/fd86cb0f4486697963b074017635cce1a6b7a4a1)). The changes is to use the license header of Spark-avro (as shown in [one of their files](https://github.com/databricks/spark-avro/blob/branch-3.2/src/main/scala/com/databricks/spark/avro/AvroOutputWriter.scala)).

**Comment by weiqingy** [comment]
Updated the license header of one of avro test files to use the license header of Spark-avro: https://github.com/hortonworks-spark/shc/pull/182/commits/67cf2a90a95b0b0eca2da2ba0ed30f8a8a39d34d

**Comment by weiqingy** [comment]
@dongjoon-hyun I have updated the PR based on current comments from Jim. Wait for his final confirmation. If this look good to him, I'll merge this.

**Pull request merged**